### ISSUE TITLE
create a register page

### ISSUE THREAD

**Issue by OlivierJM** [SparkEdUAB/sparked-mobile#7]
Just like the login page here #5 users should be allowed to skip these pages to access public info with no authentication.

**Comment by OlivierJM** [comment]
@ikayz can you take up this? any help you need I can provide

**Comment by ikayz** [comment]
Sure thing, I was trying to install React Native but kept getting errors. I'll try again today.

**Comment by OlivierJM** [comment]
Feel free to share the specific errors you are having here, maybe we can help, I remember @dennis501 had a gradle error but he fixed check in here #1

**Comment by ikayz** [comment]
@OlivierJM I managed to get React Native running. All I needed to do was run the project in Android Studio first and let it install all the dependencies

**Comment by OlivierJM** [comment]
Yeah that always works. remember we will use nativebase.io be sure to check that.

**Comment by OlivierJM** [comment]
Heyy @ikayz how is it going ? Please let me know if you need help on this

**Comment by ikayz** [comment]
Hi, I forked the project already. Working on it this weekend

**Comment by ikayz** [comment]
Sure thing, I'll do so 😎

**Comment by OlivierJM** [comment]
awesome, thanks a lot @ikayz

**Comment by ikayz** [comment]
@OlivierJM can this issue also have hacktoberfest like the others as a label?

**Comment by ikayz** [comment]
@OlivierJM Native base was not included as a dependency, added and linked.

**Comment by OlivierJM** [comment]
Hi @ikayz I will add the label as well.

**Comment by ikayz** [comment]
@OlivierJM Yes, I managed to add it. Followed the guidelines link which was shared.

**Comment by OlivierJM** [comment]
Awesome.

**Comment by ikayz** [comment]
@OlivierJM any other issue, I can work on?

**Comment by OlivierJM** [comment]
@ikayz thanks a lot, I will appreciate a lot if you can take some time to review this #14 and maybe this too #19 and maybe unblock them so that they are merged. 
There was an issue with setting up jest so we are disabling it for now until we figure it out.

**Comment by OlivierJM** [comment]
Since that is blocking other stuff once it is out then we will create more issues.

**Comment by ikayz** [comment]
Cheers will check it out

**Comment by OlivierJM** [comment]
Thanks alot.

**Comment by ikayz** [comment]
> @ikayz thanks a lot, I will appreciate a lot if you can take some time to review this #14 and maybe this too #19 and maybe unblock them so that they are merged.
> There was an issue with setting up jest so we are disabling it for now until we figure it out.

Hi Olivier, in Onboarding.js it was causing an error when the app is run due to referencing an asset with a different extension. I've fixed that on my fork.

**Comment by OlivierJM** [comment]
Ahhhh sounds great!! you can push your changes when you can and maybe merge them with this #8

**Comment by ikayz** [comment]
Cool, let me do that now and see if the tests pass

**Comment by OlivierJM** [comment]
Infact I am reopening this if you can continue from here

**Comment by ikayz** [comment]
Awesome, I've made a PR and all tests passed

**Comment by OlivierJM** [comment]
I have approved it and added some comments
